corehq/pillows/migrations/0001_elastic_v5_mapping_compatibility.py

```diff
@@ -0,0 +1,12 @@
+from django.db import migrations
+
+from ..utils import get_all_expected_es_indices, update_mapping_migration_op
+
+
+class Migration(migrations.Migration):
+
+    dependencies = []
+
+    operations = [
+        update_mapping_migration_op(info) for info in get_all_expected_es_indices()
```

Comment:
I think it would be better to hard-code the values of `CANONICAL_NAME_INFO_MAP` here rather than using a utility function since it may not be obvious to future maintainers how changing that function or the contents of `CANONICAL_NAME_INFO_MAP` will affect this migration. For example, if a new item is added to `CANONICAL_NAME_INFO_MAP` long after this migration is created, should this migration process that item? Probably not, and a new migration will be needed for where this migration has already been applied anyway.

This means adding lines like the following to this migration:
```py
from ..mappings.app_mapping import APP_INDEX_INFO
from ..mappings.case_mapping import CASE_INDEX_INFO
```
and then changing this to something like
```py
    operations = [
        update_mapping_migration_op(APP_INDEX_INFO),
        update_mapping_migration_op(CASE_INDEX_INFO),
        ...
```

Reply:
Yeah, this totally dodged my radar. Thanks for pointing it out.